vscode-extension/.eslintrc.json

```diff
@@ -9,7 +9,6 @@
         "@typescript-eslint"
     ],
     "rules": {
-        "@typescript-eslint/class-name-casing": "warn",
         "@typescript-eslint/semi": "warn",
         "curly": "warn",
         "no-throw-literal": "warn",
```

Comment:
I've been thinking there are not enough linting rules, please add an "extends" property into `.eslintrc.json` -- it would be similar to https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#configuration

Also, `"semi": "off"` can be removed because it's overridden by `"@typescript-eslint/semi": "warn"`